sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -271,7 +271,12 @@ class MPTReporter implements Reporter {
       this.testResultBatch.add(testResultObject);
       // Store test attachments in array
       const testAttachments: string[] = [];
+      const regionAndSessionIdAttachment: any[] = [];
       for (const attachment of result.attachments) {
+        if (attachment.name === "Region" || attachment.name === "SessionId") {
```

Comment:
Why limit it to Region and SessionId? Plus let's not have such specific code in the client package, if there's any changes in the backend, there's no way to fix it for older packages.